modules/tinymce/CHANGELOG.md

```diff
@@ -70,6 +70,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Creating a list in a table cell when the caret is in front of an anchor element would not properly include the anchor in the list. #TINY-6853
 - Formatting could be applied or removed on noneditable list items inside a noneditable root. #TINY-9563
 - Annotation would not be removed if the annotation was immediately deleted after being created. #TINY-9399
+- Inline dialog position is now correct even if the editor is not inline and it is contained in a `fixed`/`absolute` positioned element. #TINY-9554
```

Comment:
Every time I use / people tell me to use or since it easier to understand in various language contexts.

```suggestion
- Inline dialog position was not correct when the editor wasn't inline and was contained in a `fixed` or `absolute` positioned element. #TINY-9554
```

Reply:
done